shared/actions/config/index.js

```diff
@@ -430,6 +430,35 @@ const newNavigation = (_, action) => {
   n && n.dispatchOldAction(action)
 }
 
+function* criticalOutOfDateCheck() {
+  // check every hour
+  while (true) {
+    try {
+      const s = yield* Saga.callPromise(RPCTypes.configGetUpdateInfo2RpcPromise, {})
+      let status = 'ok'
+      let message = ''
+      switch (s.status) {
+        case RPCTypes.configUpdateInfoStatus2.ok:
+          break
+        case RPCTypes.configUpdateInfoStatus2.suggested:
+          status = 'suggested'
+          message = s.suggested?.message
+          break
+        case RPCTypes.configUpdateInfoStatus2.critical:
+          status = 'critical'
+          message = s.critical?.message
```

Comment:
Does this make message `null` if `critical` is `null`? What is it called / what can I google to learn more about it?

Reply:
It'll be undefined

Reply:
Is this part of es6? How do I google it?

Reply:
https://babeljs.io/docs/en/babel-plugin-proposal-optional-chaining